### PR TITLE
Tab stats

### PR DESCRIPTION
**Pull request by valaises**

### CODE REVIEW

File: self_hosting_machinery/webgui/selfhost_fastapi_completions.py [hunk 1]
```diff
@@ -254,7 +254,8 @@ async def _coding_assistant_caps(self):
             "cloud_name": "Refact Self-Hosted",
             "endpoint_template": "v1/completions",
             "endpoint_style": "openai",
-            "telemetry_basic_dest": "",
+            "telemetry_basic_dest": "http://localhost:8008/v1/telemetry-basic",
+            "telemetry_corrected_snippets_dest": "http://localhost:8008/v1/telemetry-snippets",
```
Comment (olegklimov) [inline self_hosting_machinery/webgui/selfhost_fastapi_completions.py:258]: remove "http://localhost:8008"

**Review by olegklimov — APPROVED** [review]

File: self_hosting_machinery/webgui/static/index.html [hunk 2]
```diff
@@ -81,6 +82,8 @@
 
   <div class="container-xxl mt-5">
     <div class="main-tab-content">
+      <div class="main-tab-pane" id="stats">
+      </div>
```
Comment (mitya52) [inline self_hosting_machinery/webgui/static/index.html:86]: I thought that we add it dynamically. Why we need to add it here?

File: self_hosting_machinery/webgui/static/style.css [hunk 3]
```diff
@@ -763,4 +763,39 @@ h3 {
     100%{
         transform: translateX(100%);
     }
-}
\ No newline at end of file
+}
```
Comment (mitya52) [inline self_hosting_machinery/webgui/static/style.css:766]: Is there any way to inherit this file? without modifications

**Review by mitya52 — CHANGES_REQUESTED** [review]

File: self_hosting_machinery/webgui/static/index.html (hunk 2 above)
Comment (valaises) [reply self_hosting_machinery/webgui/static/index.html:86]: it's not. Take a look how other tabs are defined
```
  <div class="container-xxl mt-5">
    <div class="main-tab-content">
      <div class="main-tab-pane" id="stats">
      </div>
      <div class="main-tab-pane" id="model-hosting">
      </div>
      <div class="main-tab-pane" id="upload">
      </div>
      <div class="main-tab-pane" id="finetune">
      </div>
      <div class="main-tab-pane" id="server-logs">
      </div>
      <div class="main-tab-pane" id="access-control">
      </div>
      <div class="main-tab-pane" id="settings">
      </div>
      <div class="main-tab-pane" id="license">
      </div>
    </div>
  </div>

```

File: self_hosting_machinery/webgui/static/style.css (hunk 3 above)
Comment (valaises) [reply self_hosting_machinery/webgui/static/style.css:766]: .css file must be attached in a head. All what all tabs do is insert their HTML into index.html. Technically you can create a dedicated file and attach it into index.html, but we haven't done it before

**Review by mitya52 — APPROVED** [review]

**Pull request merged**